install: document hypervisor tools (#121)
NethServer/nethsecurity-docs: install.rst

- ``<source_image.raw>`` with the actual path to your raw disk image
- ``<destination_image.vmdk>`` with your desired .vmdk filename

VMware open-vm-tools
^^^^^^^^^^^^^^^^^^^^

> VMware open-vm-tools is not part of the NethSecurity image, but it can be installed from the command line.

Reviewer: ```suggestion
VMware open-vm-tools are not part of the NethSecurity image but can be installed from the command line.
```
